src/pages/using-spark/components/button.mdx

```diff
@@ -198,28 +187,80 @@ Use when data is being saved or submitted. The spinning indicator will replace t
   titleAttr="Spinning Tertiary Button Example"
 />
 
+<SprkDivider element="span" additionalClasses="sprk-u-Measure"></SprkDivider>
 
-### Full Width at Smaller Viewports
+## Button Groups
 
-At small viewport sizes, a Button may be used at full width.
+Buttons can be grouped with other buttons to establish a relationship between them.
 
-- Stacked Buttons should have the primary action above secondary actions.
-- Use full width Buttons in containers that take up 1/3 of the grid or smaller.
+- Group buttons based on their usage and importance.
+- A max of two buttons should be in a group.
+  - A Primary button can be grouped with any less visual weight (Secondary or Tertiary) buttons.
+  - Do not mix a primary, secondary, and tertiary buttons in the same button group.
```

Comment:
What does this mean? This is implying that I can't mix any of them together.
Remove line or clarify the guidance.